### DISCUSSION TITLE
¿Junction deviation settings?

### DISCUSSION THREAD

**Discussion by Eliminateur** [supermerill/SuperSlicer#1757] [Q&A] [answered]
Hello,
i'm running marlin on my printer and i'm using junction deviation instead of regular jerk values.
But on superslicer(2.3.56a) in the machine limits, it asks for jerk limits.
so, ¿how do i either disable ONLY the jerk limits on SS or set it so it understands junction deviation?.
Or i just leave it as "use also for time estimate" or simply disable all limits?

**Comment by supermerill** [comment, marked as answer]
As long as you don't set 'Also emit limits to gcode', it's harmless. Ensure it's roughly the same as your printer, so you have a good time estimation.
